packages/core/src/components/ozon-content/readme.md

```diff
@@ -0,0 +1,33 @@
+# `<dso-ozon-content>`
+
+## Known limitations
+* Geen sluitenknop in Description component.
```

Comment:
Waarom dan? Hier is ie wel -> https://dso-toolkit.nl/_1271-description-note/components/detail/description.html